src/components/App.tsx

```diff
@@ -1028,6 +1028,12 @@ export class App extends React.Component<any, AppState> {
       });
     };
 
+    const deselectElements = () => {
+      this.setState({ selectedElementIds: {} });
+    };
+
+    deselectElements();
```

Comment:
Why do you create a function and just call it right away? Can you just call setState to deselect all the elements?

Reply:
+1, otherwise lgtm!

Reply:
Oh just thought it would make it semantically a bit better like [here](https://github.com/excalidraw/excalidraw/blob/master/src/components/App.tsx#L1024-L1029). Nothing specific though.

Just made the change and removed the extra declaration of the func.